src/client-s/css/admin-bar.scss

```diff
@@ -1,3 +1,11 @@
+@import '../../vendor/package/bourbon/bourbon-v4.2.3/bourbon';
```

Comment:
@renzms This line and the 3 that follow appear to be duplicated again below `/*![pro strip-from='lite']*/`; I suggest removing the duplicate lines and simply adjusting where `/*![pro strip-from='lite']*/` appears.